projects/js-packages/publicize-components/src/components/scheduled-posts/item-style.module.scss

```diff
@@ -14,6 +14,9 @@
 
 	.display-name {
 		text-overflow: ellipsis;
+		max-width: 70%;
+		overflow: hidden;
+		white-space: nowrap;
```

Comment:
This is to fix the this styling issue where the name was wrapped to next line

| BEFORE | AFTER |
|--------|--------|
| <img width="284" alt="Screenshot 2025-03-11 at 5 51 28 PM" src="https://github.com/user-attachments/assets/b9109e9b-753e-416a-a09d-c7177f3cd7d2" /> | <img width="311" alt="Screenshot 2025-03-11 at 5 50 44 PM" src="https://github.com/user-attachments/assets/f080918a-5c8c-4b32-82de-c37ee616d2ed" /> |